<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Receiver tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QWidget" name="left_widget" native="true">
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>461</width>
           <height>207</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="left_formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="inputDeviceLabel">
            <property name="text">
             <string>Input Device:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inputDeviceEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="audioCodecLabel">
            <property name="text">
             <string>Audio Codec:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="audioCodecEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="fileContainerLabel">
            <property name="text">
             <string>File Container</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="fileContainerEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="sampleRateLabel">
            <property name="text">
             <string>Sample Rate:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="sampleRateEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="channelLabel">
            <property name="text">
             <string>Channels: </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="channelsEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QWidget" name="empty_widget" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <widget class="Line" name="line">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>358</width>
               <height>3</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>2</height>
              </size>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="top_widget" native="true">
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>991</width>
           <height>331</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="audioTab">
          <attribute name="title">
           <string>Audio</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>4</x>
             <y>4</y>
             <width>981</width>
             <height>291</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="waveVerticalLayout"/>
          </widget>
         </widget>
         <widget class="QWidget" name="wavTab">
          <attribute name="title">
           <string>WAV</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="mp3Tab">
          <attribute name="title">
           <string>MP3</string>
          </attribute>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="right_widget" native="true">
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>481</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_left">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>120</height>
             </size>
            </property>
            <property name="title">
             <string>Encoding Mode</string>
            </property>
            <widget class="QRadioButton" name="qualityRadioButton">
             <property name="geometry">
              <rect>
               <x>21</x>
               <y>41</y>
               <width>112</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Quality</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="bitrateRadioButton">
             <property name="geometry">
              <rect>
               <x>21</x>
               <y>78</y>
               <width>112</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Bitrate</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_4" native="true">
            <widget class="QSlider" name="volumeSlider">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>31</width>
               <height>101</height>
              </rect>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
            <widget class="QLabel" name="VolumeLabel">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>50</y>
               <width>67</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>40</height>
             </size>
            </property>
            <widget class="Line" name="line_2">
             <property name="geometry">
              <rect>
               <x>-90</x>
               <y>10</y>
               <width>461</width>
               <height>20</height>
              </rect>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="widget_3" native="true">
            <widget class="QWidget" name="horizontalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>171</width>
               <height>71</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="levelsLayout"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit(&amp;E)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuExit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close(&amp;C)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
